tests/nested/core-revert/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: core revert test
+
+systems: [ubuntu-16.04-64]
```

Comment:
I think so, in fact the interesting part happens on the nested machine, if we can install qemu all should be good. I'll try locally with debian/fedora anyway.